src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
+$cell-width: 100px;
+$cell-height: 100px;
+$days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), `justify-content` and `align-items` properties should be declared within `.calendar` class or its elements since they are related to flexbox layout. The `body` tag is not using flexbox in this context.